packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/invoke-model.ts

```diff
@@ -102,9 +132,23 @@ export interface BedrockInvokeModelProps extends sfn.TaskStateBaseProps {
    * If you specify this field, the API response body is replaced with a reference to the
    * output location.
    *
-   * @default The API response body is returned in the result.
+   * @default - The API response body is returned in the result.
    */
   readonly output?: BedrockInvokeModelOutputProps;
+
+  /**
+   * The guardrail is applied to the invocation
+   *
+   * @default - No guardrail is applied to the invocation.
+   */
+  readonly guardrailConfiguration?: GuardrailConfiguration;
+
+  /**
+   * Specifies whether to enable or disable the Bedrock trace.
+   *
+   * @default - Trace is not enabled for the invocation.
+   */
+  readonly trace?: Trace;
```

Comment:
```suggestion
  readonly traceEnabled?: boolean;
```
We could avoid declaring the extra `enum`.